autopush/tests/test_endpoint.py

```diff
@@ -1401,6 +1401,8 @@ def test_base_tags(self):
                                 host='example.com:8080')
         tags = self.reg.base_tags()
         eq_(tags, ['user_agent:test', 'host:example.com:8080'])
+        tags = self.reg.base_tags()
+        eq_(tags, ['user_agent:test', 'host:example.com:8080'])
```

Comment:
The addition to the test shows the underlying issue with _base_tags being a class var -- it would fail with:

```
  File "/home/pjenvey/src/moz/autopush/autopush/tests/test_endpoint.py", line 1405, in test_base_tags
    eq_(tags, ['user_agent:test', 'host:example.com:8080'])
  File "/home/pjenvey/src/moz/autopush/site-packages/nose/tools/trivial.py", line 29, in eq_
    raise AssertionError(msg or "%r != %r" % (a, b))
AssertionError: ['user_agent:test', 'host:example.com:8080', 'user_agent:test', 'host:example.com:8080'] != ['user_agent:test', 'host:example.com:8080']
```
Here's a good explanation of the issue:
http://henrybaxter.blogspot.com/2009/11/python-and-mutable-class-variables.html